app/packages/looker/src/lookers/video.ts

```diff
@@ -1,214 +1,32 @@
-import { v4 as uuid } from "uuid";
+import { setFrameNumberAtom } from "@fiftyone/playback";
+import { getDefaultStore } from "jotai";
 import { getVideoElements } from "../elements";
 import { VIDEO_SHORTCUTS } from "../elements/common";
+import { getFrameNumber } from "../elements/util";
 import { ClassificationsOverlay, loadOverlays } from "../overlays";
-import { Overlay } from "../overlays/base";
+import type { Overlay } from "../overlays/base";
 import processOverlays from "../processOverlays";
-import {
-  BaseConfig,
-  BufferRange,
+import type {
   Buffers,
-  Coloring,
-  CustomizeColor,
-  DEFAULT_VIDEO_OPTIONS,
-  FrameChunkResponse,
   FrameSample,
   LabelData,
-  StateUpdate,
   VideoConfig,
   VideoSample,
   VideoState,
 } from "../state";
-import { addToBuffers, createWorker, removeFromBuffers } from "../util";
-
-import { setFrameNumberAtom } from "@fiftyone/playback";
-import { Schema } from "@fiftyone/utilities";
-import { getDefaultStore } from "jotai";
-import { LRUCache } from "lru-cache";
-import { CHUNK_SIZE, MAX_FRAME_CACHE_SIZE_BYTES } from "../constants";
-import { getFrameNumber } from "../elements/util";
+import { DEFAULT_VIDEO_OPTIONS } from "../state";
+import { addToBuffers, removeFromBuffers } from "../util";
 import { AbstractLooker } from "./abstract";
-import { LookerUtils } from "./shared";
-
-interface Frame {
-  sample: FrameSample;
-  overlays: Overlay<VideoState>[];
-}
-
-type RemoveFrame = (frameNumber: number) => void;
-
-interface AcquireReaderOptions {
-  addFrame: (frameNumber: number, frame: Frame) => void;
-  addFrameBuffers: (range: [number, number]) => void;
-  removeFrame: RemoveFrame;
-  getCurrentFrame: () => number;
-  dataset: string;
-  group: BaseConfig["group"];
-  view: any[];
-  sampleId: string;
-  frameNumber: number;
-  frameCount: number;
-  update: StateUpdate<VideoState>;
-  dispatchEvent: (eventType: string, detail: any) => void;
-  coloring: Coloring;
-  customizeColorSetting: CustomizeColor[];
-  schema: Schema;
-}
-
-const { acquireReader, addFrame } = (() => {
-  const createCache = () =>
-    new LRUCache<WeakRef<RemoveFrame>, Frame>({
-      maxSize: MAX_FRAME_CACHE_SIZE_BYTES,
-      sizeCalculation: (frame) => {
```

Comment:
I chose to remove LRU caching based on item size. Based on performance reports, it caused more problems than it solved. 1000 frames is now the default cache size, which can be figured via `FIFTYONE_APP_FRAME_STREAM_SIZE`